lib/AST/Type.cpp

```diff
@@ -3388,8 +3388,8 @@ AnyFunctionType::ExtInfo::assertIsFunctionType(const clang::Type *type) {
     SmallString<256> buf;
     llvm::raw_svector_ostream os(buf);
     os << "Expected a Clang function type wrapped in a pointer type or "
-       << "a block pointer type but found:\n";
-    type->dump(os);
+       << "a block pointer type but found:\n <TODO>";
+    // type->dump(os);
```

Comment:
@varungandhi-apple getting a clang ASTContext through to here seemed a little more involved than the others. Are you ok with me filing an issue for you (or someone else) to look into fixing this one as a follow-up?

Reply:
Sure, that's sounds reasonable since this is a debug statement. I can follow up.

Reply:
Thanks!